cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -8,45 +8,46 @@ toc::[]
 [id="odo-notable-improvements_{context}"]
 == Notable changes and improvements in `{odo-title}`
 
-* A `--now` flag is added for `odo create`, `odo url`, and `odo config`. 
-+
-Run `odo url create --now` to create the URL in the configuration and propagate all the changes to the cluster.
-* An `odo debug info` command is added. The command displays if the debug mode is enabled for a component, if the port-forward process is running, and information about ports used.
-* The `odo push` output now shows the exact error from the OpenShift cluster on failure.
-+
-* A `--secure` flag is added for `odo url`. It indicates whether a URL is secure. 
-* `odo storage list` now displays the information about the storage state: `Pushed`, `NotPushed` or `Locally Deleted`.
-* `odo debug port-forward` now selects a port automatically if the default port (5858) is occupied.
-* The `--all` flag is renamed to  `--all-apps`.
-* The default PVC size has been increased to 10GiB.
-* The JSON output for `odo storage list -o json` has been restructured.
-* `nodejs-8` and `nodejs-10` images are no longer supported.
-* Experimental mode feature. By default, features under development or in experimental mode are hidden from the user. 
-+
-** To enable experimental mode, run:
-+
-----
-$ odo config set --env ODO_EXPERIMENTAL=true
-----
-+
-or
-+
-----
-$ odo preference set experimental true
-----
-** To disable experimental mode, run:
-+
-----
-$ odo config unset --env ODO_EXPERIMENTAL
-----
-+
-or
+* The `--devfile` flag is added to `odo create`. Run `odo create <component name> --devfile <devfile path>` to specify your devfile location.
+
+* Dynamic registry support. Now you can configure your own registries with the following commands:
 +
 ----
-$ odo preference set experimental false
+# Add registry to the registry list
+odo registry add <registry name> <registry URL>
+
+# List registry in the registry list
+odo registry list
+
+# Delete registry from the registry list
+odo registry delete <registry name>
+
+# Update registry in the registry list
+odo registry update <registry name> <registry URL>
+
+# List component with corresponding registry
+odo catalog list components
+
+# Create component that is hosted by specific registry
+odo create <component type> --registry <registry name>
 ----
 
-* `{odo-title}` now supports Ingress to create URLs on Kubernetes.
+* The `--downloadSource` flag is added to `odo create`. Run `odo create nodejs --downloadSource <project-name>` to download a source code of a project specified in the devfile. If no project name is specified, `odo` downloads the first one.
```

Comment:
This is actually being renamed to `--starter` in this PR: https://github.com/openshift/odo/pull/3425

Should we still have this in the release notes?

Reply:
Thanks! I've renamed it to `--starter`. I see that the PR has been approved, any reasons it might not get merged and not make it into the release? If so, we can leave it out.